### PR TITLE
Request a tag buttons on interest/skills pages (md 1537a)

### PR DESCRIPTION
**Pull request by a-pasquale**
## Describe context / purpose for this PR
Add button to “Add Interests” and “Add Expertise” pages linking to form to request a new tag.

## Issue link
https://cyberteamportal.atlassian.net/browse/D8-1537

## Any other related PRs?
n/a

## Link to MultiDev instance
http://md-1537a-accessmatch.pantheonsite.io

## Checklist for PR author
- [ ] I have checked that the PR is ready to be merged
- [ ] I have reviewed the DIFF and checked that the changes are as expected
- [ ] I have assigned myself or someone else to review the PR

### CODE REVIEW

**Comment by laheara143** [comment]
@a-pasquale I fixed the request tag on nect, campus champion, and ACCESS theme. I also fixed the description layout on nect and campus champion. There seems to be a conflict with the composer.lock but I am not sure which is the most up to date.

**Comment by a-pasquale** [comment]
The block is broken or missing:
image
Are you placing this block? /block/8 or creating a new one?

**Comment by laheara143** [comment]
> The block is broken or missing: image Are you placing this block? /block/8 or creating a new one?

That should be the "request tag" button. I created a new block for that @a-pasquale

**Comment by a-pasquale** [comment]
I think you should be able to use the existing custom block, linked in the prior comment. Is there a reason to create another one?

**Comment by laheara143** [comment]
> I think you should be able to use the existing custom block, linked in the prior comment. Is there a reason to create another one?



> I think you should be able to use the existing custom block, linked in the prior comment. Is there a reason to create another one?

That block contains the Tree View and List View button so I though it would be best to add a separate block with just the request tag button  
image

File: web/sites/default/config/default/block.block.skillsdescription.yml [hunk 1]
```diff
@@ -6,15 +6,14 @@ dependencies:
...
-    - domain
     - system
     - user
   theme:
     - nect
 id: skillsdescription
 theme: nect
-region: content
-weight: 8
+region: title_area
```
Comment (laheara143) [inline web/sites/default/config/default/block.block.skillsdescription.yml:15]: @a-pasquale I already fixed the descriptions on this branch but will I need to do it on a new branch in order to export it?

**Comment by laheara143** [comment]
Closed and remade in PR #891

**Pull request closed without merging**